ToNote/Controls/NotePanel.cs

```diff
@@ -11,9 +11,9 @@
     using ToNote.Logic;
     using ToNote.Models;
 
-    public class NoteRichTextBoxPanel : ItemsControl
+    public class NotePanel : ItemsControl
     {
-        public NoteRichTextBoxPanel()
+        public NotePanel()
         {
             // CTRL + Arrow Up or CTRL + Arrow Down navigates to a textbox that is above or below currently focused one, respectively.
```

Comment:
Update comment, `to a textbox` -> `to an item` or `to a textbox or todo item` or similar